pnpm-lock.yaml

```diff
@@ -74,7 +74,7 @@ importers:
       '@astrojs/sitemap': ^1.0.0
       astro: ^2.0.0-beta.3
     dependencies:
-      '@astrojs/mdx': link:../../packages/integrations/mdx
+      '@astrojs/mdx': 1.0.0-beta.2_astro@packages+astro
```

Comment:
I think this incorrect `@astrojs/mdx` changes should be fixed once you merged up `main` into this branch and run `pnpm i` again. Otherwise maybe it's fine to leave it as is as CI would update the lockfile again (not quite sure).